source/upgrade/upgrading-mattermost-server.rst

```diff
@@ -18,6 +18,34 @@ In most cases, you can upgrade Mattermost Server in a few minutes. However, the
 Prepare to upgrade to the latest version
 ------------------------------------------
 
+Upgrade to Mattermost v7.1
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Mattermost v7.1 introduces schema changes in the form of a new column and its index. The following notes our test results for the schema changes:
+
+- MySQL 12M Posts, 2.5M Reactions - ~1min 34s (instance: PC with 8 cores, 16GB RAM)
+- PostgreSQL 12M Posts, 2.5M Reactions - ~1min 18s (instance: db.r5.2xlarge)
+
+Customers wanting an upgrade with substantially reduced downtime (~1/5th of the original time) are encouraged to run the following queries before upgrading. This is fully backwards-compatible, but will obtain a lock on ``Reactions`` table, so users' reactions posted during this time won't be reflected in the database.
```

Comment:
Can you clarify from where you got the info on reduced downtime? The downtime will be the same however way the upgrade is done. This is just to facilitate the customer to easily schedule a maintenance operation without needing to upgrade Mattermost.

Reply:
@agnivade - See [this conversation](https://community.mattermost.com/private-core/pl/3yfqzushmff3b8ekx99w3h66xe) for details.

Reply:
Hey Agniva, Carrie, not sure if I put the text correctly.

What I meant is, pre-running a migration only blocks Reactions table, and other functionalities will work just fine. The update query on Reactions takes 4/5th of the migration time.

If mm-server start is blocked by migrations, then no functionality is available for the full migration time.

Hope I clarified things.

Reply:
Thanks Shivashis. Carrie - let's change the sentence to "Customers wanting to run the SQL queries before-hand can run these queries".

The fact that the migrations will obtain a lock can be mentioned at the beginning.

> so users' reactions posted during this time won't be reflected in the database.

This is not entirely correct. The reactions will be posted, but it will be blocked until the migrations are complete.

I think we should decide on a fixed format for these migration release notes. We do the same thing for every release. It might be helpful to be consistent every time rather than come up with different text for each release.

Reply:
Thanks, @plant99 -- the interpretation error is all mine. I appreciate your help clarifying the message we want to communicate!

Reply:
@cwarnermm - Feel free to let me know if you have any questions. Happy to help :)